.github/containerscan/.trivyignore

```diff
@@ -0,0 +1 @@
+# Add ignored CVEs in new lines below
```

Comment:
We can add CVE-2022-3171 with reference to https://github.com/hazelcast/hazelcast/issues/20807 and https://issues.apache.org/jira/browse/HADOOP-18197